app/src/main/java/com/github/onlynotesswent/ui/user/EditProfile.kt

```diff
@@ -217,47 +248,118 @@ fun EditableProfilePicture(
+                .size(40.dp)
+                .background(
+                    androidx.compose.material3.ButtonDefaults.buttonColors()
+                        .containerColor) // Background color
+        ,
+        content = {
           Icon(
               imageVector = Icons.Default.Edit,
-              contentDescription = "Edit Profile Picture",
-              modifier = Modifier.testTag("editProfilePicture").size(40.dp) // Size of the edit icon
-              ,
-              tint = Color.Blue // Icon color
-              )
-        }
+              contentDescription = "Display Bottom Sheet",
+              modifier = Modifier.size(30.dp),
+              tint =
+                  androidx.compose.material3.ButtonDefaults.buttonColors()
+                      .contentColor
+                      .copy(alpha = 0.8f))
+        })
 
-    // Delete Image Icon
-    IconButton(
-        onClick = { // Delete the image and clear the uri
-          profilePictureUri.value = ""
-          hasProfilePictureBeenChanged.value = true
-        },
-        modifier =
-            Modifier.align(Alignment.CenterEnd) // Position on the bottom-left corner
-                .offset(x = 42.dp, y = (-25).dp)
-                .clip(CircleShape)) {
-          Icon(
-              imageVector = Icons.Default.Delete,
-              contentDescription = "Delete Profile Picture",
-              modifier =
-                  Modifier.testTag("deleteProfilePicture").size(40.dp), // Size of the edit icon
-              tint = Color.Red // Icon color
-              )
+    if (showSheet.value) {
+      ModalBottomSheet(onDismissRequest = { showSheet.value = false }, sheetState = sheetState) {
+        BottomSheetContent(
+            onClose = { showSheet.value = false },
+            profilePictureTaker,
+            profilePictureUri,
+            hasProfilePictureBeenChanged)
+      }
+    }
+  }
+}
+
+@Composable
+fun BottomSheetContent(
+    onClose: () -> Unit,
+    profilePictureTaker: ProfilePictureTaker,
+    profilePictureUri: MutableState<String>,
+    hasProfilePictureBeenChanged: MutableState<Boolean>
+) {
+  Column(
+      modifier = Modifier.fillMaxWidth().padding(30.dp),
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.Center) {
+        if (profilePictureUri.value.isNotBlank()) {
+          // If the user has a profile picture, display the edit and remove options
+          Column(horizontalAlignment = Alignment.Start) {
+            BottomSheetRow(
+                {
+                  profilePictureTaker.setOnImageSelected { uri ->
+                    if (uri != null) {
+                      profilePictureUri.value = uri.toString()
+                      hasProfilePictureBeenChanged.value = true
+                    }
+                  }
+                  profilePictureTaker.pickImage()
+                  onClose()
+                },
+                "Edit Profile Picture",
```

Comment:
I suggest capitalizing only the first word, to be consistent with the rest of the project